cpp/include/cudf/utilities/chrono.hpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <simt/chrono>
+#include <simt/../details/iterator>
```

Comment:
These imports split out into a separate header so we can't accidentally include `simt/chrono` without `details/iterator`, which causes compiler issues in certain cases.